soh/soh/OTRGlobals.h

```diff
@@ -65,6 +65,22 @@ class OTRGlobals
 uint32_t IsGameMasterQuest();
 #endif
 
+#define RANDOMIZER_CVAR(var) "gRandoEnhancements." var
+#define RANDOMIZER_SEED_CVAR(var) "gRandoSettings." var
+#define COSMETIC_CVAR(var) "gCosmetics." var
+#define AUDIO_CVAR(var) "gAudioEditor." var
+#define REPL_SEQ_CVAR(var) AUDIO_CVAR("ReplacedSequences." var)
+#define EXCL_SEQ_CVAR(var) AUDIO_CVAR("Excluded." var)
```

Comment:
```suggestion
#define REPLACE_SEQ_CVAR(var) AUDIO_CVAR("ReplacedSequences." var)
#define EXCLUDE_SEQ_CVAR(var) AUDIO_CVAR("Excluded." var)
```

i think the extra 3 characters are worth it for clarity here, i'm kind of torn on if it makes sense to go all the way and have `REPLACE_SEQUENCE_CVAR` instead.

Reply:
This makes sense, since I was so verbose with most of the other macros.